sonar-project.properties

```diff
@@ -1,8 +1,7 @@
-sonar.coverage.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
-sonar.cpd.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
-sonar.exclusions=.vscode-test/**/*,Gulpfile.mjs,bin/**/*,mk-files/**/*,node_modules/**/*,playwright.config.ts,src/**/*.spec.ts,src/**/*.test.ts,src/clients/**/*,src/testing.ts,tests/**/*,userdata/**/*
-sonar.javascript.lcov.reportPaths=coverage/lcov-functional.info,coverage/lcov.info
-sonar.language=javascript,typescript
 sonar.projectKey=vscode
+sonar.language=javascript,typescript
 sonar.sources=src
-sonar.typescript.coverage.reportPaths=coverage/lcov-functional.info,coverage/lcov.info
+sonar.javascript.lcov.reportPaths=coverage/lcov-functional.info,coverage/lcov.info
+sonar.coverage.exclusions=src/testing.ts,src/**/*.test.ts,src/clients/**/*,src/**/*.spec.ts
+sonar.cpd.exclusions=src/testing.ts,src/**/*.test.ts,src/clients/**/*,src/**/*.spec.ts
+sonar.exclusions=src/testing.ts,src/**/*.test.ts,src/clients/**/*,src/**/*.spec.ts
```

Comment:
The fact that these are in `sonar.exclusions` means we shouldn't even see them appear for coverage/duplication analysis, but I'm keeping the `sonar.coverage.exclusions` and `sonar.cpd.exclusions` just to be safe